ft_volumerealign.m

```diff
@@ -1601,9 +1601,13 @@ function cb_redraw(h, eventdata)
 % for some unknown god-awful reason, the line command 'disables' all transparency
 % the below command resets it (it was the only axes property changed after adding the crosshair
 % I could find, and putting it back to 'childorder' instead of 'depth' fixes the problem. Lucky find -roevdmei
-set(h1,'sortMethod','childorder')
-set(h2,'sortMethod','childorder')
-set(h3,'sortMethod','childorder')
+% -update, the line command only 'disables' in the new graphics system introduced in 2014b (any version below is fine,
+% and does not contain the sortmethod property --> crash) - roevdmei
+if ~verLessThan('matlab','8.4') % 8.4 = 2014b
```

Comment:
I was very busy at the time and couldn't remember/find the fieldtrip
function we have for this purpose. Was it ft_platform_supports?

--
Roemer van der Meij, PhD
Postdoctoral Researcher
Voytek Lab
University of California, San Diego
Cognitive Science
9500 Gilman Drive
La Jolla, CA 92093-0515
E-mail: roemer.van.der.meij@gmail.com
On May 9, 2016 00:01, "Robert Oostenveld" <notifications@github.com> wrote:

> In ft_volumerealign.m
> <https://github.com/fieldtrip/fieldtrip/pull/141#discussion_r62456143>:
>
> > @@ -1601,9 +1601,13 @@ function cb_redraw(h, eventdata)
> >  % for some unknown god-awful reason, the line command 'disables' all transparency
> >  % the below command resets it (it was the only axes property changed after adding the crosshair
> >  % I could find, and putting it back to 'childorder' instead of 'depth' fixes the problem. Lucky find -roevdmei
> > -set(h1,'sortMethod','childorder')
> > -set(h2,'sortMethod','childorder')
> > -set(h3,'sortMethod','childorder')
> > +% -update, the line command only 'disables' in the new graphics system introduced in 2014b (any version below is fine,
> > +% and does not contain the sortmethod property --> crash) - roevdmei
> > +if ~verLessThan('matlab','8.4') % 8.4 = 2014b
>
> Since I was not sure that verLessThan is available on old MALTA'S, I
> checked 2011a. It is present there, so it is fine to use.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/fieldtrip/fieldtrip/pull/141/files/9a646187334529a2d919034e9965bcc0fd779be3#r62456143>
>